### PR TITLE
samples: display: add fixture

### LINKED ISSUE #28255: mimxrt1050_evk:running samples/drivers/display/ failed
Describe the bug
running samples/drivers/display/, and the result just stay displaying blanking control not implemented

To Reproduce
west build -b mimxrt1050_evk samples/drivers/display/
west flash
see error

*** Booting Zephyr OS build v2.4.0-rc1-19-gd7468bf836b7  ***
[00:00:00.006,000] <inf> sample: Display sample for ELCDIF_1
[00:00:04.527,000] <err> display_mcux_elcdif: **Display blanking control not implemented**

Environment (please complete the following information):
OS: Fedora28
Toolchain: Zephyr-sdk-0.11.3
Commit ID: d7468bf836b7

### PR DESCRIPTION
**Pull request by nashif**
Add fixture so this test only runs on devices that have a display
connected.

Fixes #28255

### CODE REVIEW

**Review by MaureenHelm — APPROVED** [review]

File: samples/drivers/display/sample.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 sample:
   description: Sample application for displays
   name: display_sample
+common:
+  harness: console
+  harness_config:
+    fixture: fixture_display
```
Comment (vanwinkeljan) [inline samples/drivers/display/sample.yaml:7]: Does this play nicely with the native_posix target, if not we will lose coverage in CI?

Comment (vanwinkeljan) [reply samples/drivers/display/sample.yaml:7]: Just double checked this change prevents execution of the test case on the native_posix board.

**Review by vanwinkeljan — CHANGES_REQUESTED** [review]
Looks like native target boards should be excluded from tes fixture checking

**Comment by nashif** [comment]
> Looks like native target boards should be excluded from tes fixture checking

yeah, need to do it per section...

**Review by vanwinkeljan — APPROVED** [review]

**Pull request merged**